docs/response-parameters.rst

```diff
@@ -1,21 +1,29 @@
 Response Parameters
 ===================
 
-Descriptions of the custom response parameters that are marked with a leading ``@`` for the 
-data extraction and contributions endpoints.
+Description of the response parameters.
 
 General Parameters
 ------------------
 
-* ``@osmId`` - id of the OSM feature with its type (node/way/relation)
-* ``@version`` - version of the feature
-* ``@changesetId``- id assigned to a changeset
-* ``@osmType``- type (node/way/relation) of OSM feature
+Description of parameters that are present in every response.
 
+* ``attribution`` - copyrights and attribution
+* ``apiVersion`` - version of the ohsome API
+
+Aggregation Parameters
+----------------------
```

Comment:
is this section empty on purpose?

Reply:
yes, as it's out of scope from this PR and should indicate that we want to include docs on the response formats of aggregation endpoints

Reply:
ok. could you please open a github issue for that (in the 1.3 milestone), so we don't forget it. :)